src/ViewModels/CkanGroupViewModel.js

```diff
@@ -76,6 +77,14 @@ var CkanGroupViewModel = function(application) {
      */
     this.minimumMaxScaleDenominator = undefined;
 
+    /**
+     * Gets or sets any extra wms parameters that should be added to the wms query urls in this CKAN group.
+     * If this property is undefined or if {@link CkanGroupViewModel#filterByWmsGetCapabilities} then no parameters are added
```

Comment:
This is so we can pass the tiled parameter on source we detect have tiled.  This is detected in getCapabilities version 1.1.1
________________________________________
From: Kevin Ring [notifications@github.com]
Sent: Tuesday, 20 January 2015 2:18 PM
To: NICTA/nationalmap
Cc: Keith Grochow
Subject: Re: [nationalmap] CKAN support for base NM Catalog (#485)

In src/ViewModels/CkanGroupViewModel.js<https://github.com/NICTA/nationalmap/pull/485#discussion-diff-23200946>:

> @@ -76,6 +77,14 @@ var CkanGroupViewModel = function(application) {
>       */
>      this.minimumMaxScaleDenominator = undefined;
>
> +    /**
> +     * Gets or sets any extra wms parameters that should be added to the wms query urls in this CKAN group.
> +     * If this property is undefined or if {@link CkanGroupViewModel#filterByWmsGetCapabilities} then no parameters are added


I'm pretty sure this sentence isn't true. wmsParameters is passed to the created WMS catalog items as their parameters property, regardless of the filterByWmsGetCapabilities property, correct?

—
Reply to this email directly or view it on GitHub<https://github.com/NICTA/nationalmap/pull/485/files#r23200946>.


________________________________

The information in this e-mail may be confidential and subject to legal professional privilege and/or copyright. National ICT Australia Limited accepts no liability for any damage caused by this email or its attachments.